src/sitegen/layout.cljs

```diff
@@ -37,7 +37,7 @@
     [:title title]
     [:meta {:name "description" :content description}]
 
-    [:link {:rel "stylesheet" :href "http://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700"}]
+    [:link {:rel "stylesheet" :href "https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700"}]
```

Comment:
Paul Irish says not to do this anymore: http://www.paulirish.com/2010/the-protocol-relative-url/

> Now that SSL is encouraged for everyone and doesn’t have performance concerns, this technique is now an anti-pattern. If the asset you need is available on SSL, then always use the https:// asset.